src/Sub.php

```diff
@@ -0,0 +1,24 @@
+<?php
+namespace Laminas\Filter;
+
+use Laminas\Filter\AbstractFilter;
+use Laminas\Filter\Exception\InvalidArgumentException;
+
+class Sub extends AbstractArithmeticOperation
+{
+    /**
+     *
+     * {@inheritdoc}
+     * @see \Laminas\Filter\FilterInterface::filter()
+     */
+    public function filter($value)
+    {
+        if (! isset($this->options['operand'])) {
+            throw new InvalidArgumentException(sprintf('%s expects argument value; received none', __METHOD__));
+        }
+
+        $value = (float) $value;
+        $operand = (float) $this->options['operand'];
```

Comment:
Same remark as for the `Add` filter; should this maybe be broken into `SubInt` and `SubFloat`?